R/pin-read-write.R

```diff
@@ -30,14 +33,34 @@
 #'
 #' # can use `version` argument to read a specific version:
 #' pin_versions(model_board, "cars_linear")
+#' @examplesIf interactive() || identical(Sys.getenv("IN_PKGDOWN"), "true")
+#' # can store an renv lockfile as part of the pin:
+#' vetiver_pin_write(model_board, v, check_renv = TRUE)
 #'
 #' @export
-vetiver_pin_write <- function(board, vetiver_model, ...) {
+vetiver_pin_write <- function(board, vetiver_model, ..., check_renv = FALSE) {
+
+    lockfile <- character(0)
+
+    if (check_renv) {
+        pkgs <- vetiver_required_pkgs(vetiver_model$metadata$required_pkgs)
```

Comment:
TODO: This should only be the packages for the model itself, not infrastructure like plumber and such.

Reply:
We decided to try doing the model packages plus vetiver only, and see how that goes.